dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -315,4 +330,118 @@ public Response saveLayout(@Context final HttpServletRequest request, final Page
         return res;
     }
 
+    /**
+     * Add a content to a specific container into the page
+     *
+     * @param req
+     * @param res
+     * @param containerId container's id
+     * @param contentletId content's id
+     * @param order order to add the content
+     * @param uid {@link MultiTree} relationship type
+     * @param pageId
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     * @throws ParseErrorException
+     * @throws MethodInvocationException
+     * @throws ResourceNotFoundException
+     * @throws IOException
+     * @throws IllegalAccessException
+     * @throws InstantiationException
+     * @throws InvocationTargetException
+     * @throws NoSuchMethodException
+     */
+    @POST
+    @JSONP
+    @NoCache
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    @Path("{pageId}/add/container/{containerId}/content/{contentletId}/uid/{uid}/order/{order}")
+    public final Response addContentToContainer(@Context final HttpServletRequest req, @Context final HttpServletResponse res,
+                                                @PathParam("containerId") final String containerId,
+                                                @PathParam("contentletId") final String contentletId,
+                                                @PathParam("order") final int order, @PathParam("uid") final String uid,
+                                                @PathParam("pageId") final String pageId)
+            throws DotDataException, DotSecurityException, ParseErrorException, MethodInvocationException,
+            ResourceNotFoundException, IOException, IllegalAccessException, InstantiationException,
+            InvocationTargetException, NoSuchMethodException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.reflect.InvocationTargetException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.NoSuchMethodException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)